src/app/routes.tsx

```diff
@@ -3,26 +3,55 @@ import { Redirect, Route, RouteComponentProps, Switch } from 'react-router-dom';
 import { DynamicImport } from '@app/DynamicImport';
 import { Dashboard } from '@app/Dashboard/Dashboard';
 import { NotFound } from '@app/NotFound/NotFound';
+import DocumentTitle from 'react-document-title';
 
 const getSupportModuleAsync = () => {
   return () => import(/* webpackChunkName: 'support' */ '@app/Support/Support');
 }
-const Support = () => {
+
+const Support = (routeProps: RouteComponentProps) => {
   return (
     <DynamicImport load={getSupportModuleAsync()}>
       {(Component: any) => {
-        return Component === null ? <p>loading</p> : <Component.Support />
+        return Component === null ? <p>loading</p> : <Component.Support {...routeProps} />
       }}
     </DynamicImport>
   );
 }
 
+const RouteWithTitleUpdates = ({
+  component: Component,
+  title,
+  ...rest
+}) => {
+  function routeWithTitle(routeProps: RouteComponentProps) {
+    routeProps.history.listen(() => {
+      // wait a moment to ensure assistive technology has had
+      // a chance to notice the new content before sending focus
+      setTimeout(() => {
+        const mainContainer = document.getElementById('primary-app-container');
+        mainContainer && mainContainer.focus();
+      }, 500);
```

Comment:
This static delay doesn't work well with async components. Looking for a way to fire this function n seconds _after_ the target route has rendered, instead of the moment when the route is first requested.

Reply:
Fixed with https://github.com/patternfly/patternfly-react-seed/pull/57/commits/2c3552d7200d7f13c18698911301ac7e0550525e#diff-253513ac6a6cd48fa4017f8a2fc21417R21-R22